frontend/components/layouts/LayoutEditor.js

```diff
@@ -55,7 +75,7 @@ export default props => {
           />
         </div>
         <div className='col-6'>
-          <iframe srcDoc={editingLayout.content} className='w-100' style={{ height: '75vh' }}>
+          <iframe srcDoc={editingLayout.content} className='w-100' style={{height: `${50*editingLayout.height/editingLayout.width}vw`}}>
```

Comment:
`w-100` here equals 50 percent visible-width (because of `col-6` ), hence `height = 50*height/width visible width`